### PR TITLE
Organization registration form, CSS corrections

### PR DESCRIPTION
**Pull request by AlisaSev**
Created a form for organization registration on LADP. Implemented Formik and Yup for form validation. Matched the CSS to the user registration form. Adjusted the heading's top and bottom margins.

image

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -11,6 +11,7 @@ import Register from "./components/register/Register";
 import NotFound from "./components/NotFound";
 import ContactUs from "./components/contact/ContactUs";
 import NavBar from "./components/navBar/NavBar";
+import RegisterOrg from "./components/register/RegisterOrg";
 
 function App() {
   const [isLoggedIn, setIsLoggedIn] = useState(true);
```
Comment (victoremartinez18) [inline src/App.jsx:17]: Raphael changed this line to set the state to false, to prevent the logout button to show when you are not logged in.

**Review by victoremartinez18 — COMMENTED** [review]
The code looks good, but if Raphael's code is merged first you will have to pull from main to have the updated App.jsx.

**Review by AlisaSev — COMMENTED** [review]
Everything looks ok.

**Review by na106 — APPROVED** [review]
The code looks good

**Review by victoremartinez18 — APPROVED** [review]

**Pull request merged**